### ISSUE TITLE
Add the green shield in the editor when activated

### ISSUE THREAD

**Issue by jasell** [ckeditor/github-writer#474]
## 📝 Add the green shield in the editor when activated

When I come to a github editor I would like to see the green shield as an indicator that the editor is CKEditor.
There are many incarnations of MarkDown editors in the github environment, currently only some are supported by CKEditor, hence it would be great to see when it is enabled.

Like the editor where we write this request is not supported by CKEditor.
---

If you'd like to see this feature implemented, add a 👍 reaction to this post.

**Comment by Drumstix42** [comment]
Agreed. Would also like to see a way to toggle the extension on and off within the page. Sometimes there are issues that require disabling the extension and reloading the page.
